<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationDialog</class>
 <widget class="QDialog" name="CalibrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_writeCalibFile">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Write Calibration to File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_readCalibFile">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Read Calibration from File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_message">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>471</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Odd (0)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Even(1)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Darks</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_darks0">
      <property name="text">
       <string>7800</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_darks1">
      <property name="text">
       <string>7800</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_constDark">
      <property name="text">
       <string>Use Constant Darks</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Gain Ratio</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_ratio0">
      <property name="text">
       <string>19.5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_ratio1">
      <property name="text">
       <string>19.5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_constRatio">
      <property name="text">
       <string>Use Constant Ratio</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Offset</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_offset0">
      <property name="text">
       <string>370</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_offset1">
      <property name="text">
       <string>370</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_constOffset">
      <property name="text">
       <string>Use Constant Offsets</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>ADU per dump</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_dADUdN0">
      <property name="text">
       <string>5900</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineEdit_dADUdN1">
      <property name="text">
       <string>5900</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_constDump">
      <property name="text">
       <string>Use Constant Dumps</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Displayed values only used to write constant arrays in calibration. Values Not read from calibration.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
